addons/bestja_account_deletion/models.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+import uuid
+
+
+class Volunteer(models.Model):
+    _inherit = 'res.users'
+    STATES = [
+        ('active', 'aktywny'),
+        ('not_activated', 'nieaktywowany'),
+        ('deleted', 'usuniety')
+    ]
+
+    active_state = fields.Selection(STATES, default='not_activated', store=True, string="Stan")
```

Comment:
You just add a new list item to the selection, you don't need to repeat everything else. This is enough:

```
active_state = fields.Selection(selection_add=[('deleted', 'usuniety')])
```